TUTORIAL_USE.md

```diff
@@ -0,0 +1,118 @@
+# Using the existing code generators
+
+Java::Geci provides several out-of-the-box code generators. 
+In this tutorial we'll use the generator `Accessor` as an example, 
+but the things explained in this tutorial apply to every other generator.
+
+You use Java::Geci by using the `Geci` class in tests.
+
+```java
+public class TestAccessor {
+
+    @Test
+    public void testAccessor() throws Exception {
+        Assertions.assertFalse(new Geci().source(maven()
+                        .module("javageci-examples").mainSource())
+                        .register(Accessor.builder().build()).generate(),
+                Geci.FAILED);
+    }
+}
+```
+This might seem like a lot at first, so let's break it down.
+
+`Geci` has a fluent interface.
+
+The method `source()` can be used to specify the directories where your
+source files are. If you have source files in different places you
+have to chain several `source()` invocations one after the other. Every
+single call to `source()` can specify several directories. These count
+as one single directory and the first that exists is used to discover
+the files.<br/>
+This means:
+* Call `source("foo").source("bar")` if you have sources in **both** foo **and** bar.
+* Call `source("foo", "bar")` if you have sources in **either** foo **or** bar.
+*maven().module("javageci-examples").mainSource()* is just "javageci-examples/src"
+
+The method `register()` can register one or more source code generators. 
+You can also chain `register()` calls and register your generators one-by-one. 
```

Comment:
I would say generator objects, since there can be different instances of the same generator and sometimes there really are. If you want to generate using repeated but using two different lists then you need to instances of repated configured so that they work under different mnemonic. That is why they have the mnemonic configurable in the builder API.

Reply:
Noted, will correct.